.gitignore

```diff
@@ -0,0 +1,3 @@
+/node_modules
+storage.js
```

Comment:
storage.js should not be in .gitignore because this file is required for the app to work properly and can't be downloaded from npm or some other places